src/vm/jitinterface.cpp

```diff
@@ -10914,27 +10939,7 @@ void CEEJitInfo::reserveUnwindInfo(BOOL isFunclet, BOOL isColdCode, ULONG unwind
 
```

Comment:
@parjong from your other PR, I have just realized you are trying to add unwind info that's a clone of the amd64 unwind info for x86 Linux. @jkotas, your comment was assuming that we are not going to introduce that for x86, right?